qp/pdf.py

```diff
@@ -96,12 +94,14 @@ def integrate(self, limits):
         """
         return None
 
-    def quantize(self, percent=1., number=None, vb=True):
+    def quantize(self, quants=None, percent=1., number=None, vb=True):
         """
         Computes an array of evenly-spaced quantiles from the truth.
 
         Parameters
         ----------
+        quants: ndarray, float
+            array of quantile locations as decimals
```

Comment:
I see - you are punting the choice of quantpoints to the user. Fine! It would be good to come up with a good algorithm for setting the quantpoints, though.